tcf_website/templates/base/sidebar.html

```diff
@@ -1,35 +1,49 @@
 {% load static %}
 
 <!-- Sidebar -->
-<div class="border-right
-d-flex flex-column justify-content-between" id="sidebar-wrapper">
+<div id="sidebar-wrapper" class="border-right pb-0
+     d-flex flex-column justify-content-between text-center">
   <div class="list-group list-group-flush">
     <!-- TODO: dashboard for saved courses, enrollment dates, major-specific info, etc -->
     <!-- <a href="{% url 'browse' %}"
        class="list-group-item list-group-item-action">
        <i class="fa fa-star-o fa-fw"></i>&nbsp; Dashboard
     </a> -->
     <a href="{% url 'browse' %}"
-       class="list-group-item list-group-item-action
+       class="list-group-item list-group-item-action pb-0
        {% if request.resolver_match.url_name == 'browse' %}active{% endif %}">
-       <i class="fa fa-th-large fa-fw"></i>&nbsp; Browse Courses
+       <h4><i class="fa fa-th-large fa-fw"></i></h4>
+       <p>Browse</p>
+       <hr>
     </a>
     <a href="{% url 'new_review' %}"
-       class="list-group-item list-group-item-action
+       class="list-group-item list-group-item-action pb-0
        {% if request.resolver_match.url_name == 'new_review' %}active{% endif %}">
-      <i class="fa fa-pencil-square-o fa-fw"></i>&nbsp; Write Review
+      <h4><i class="fa fa-pencil-square-o fa-fw"></i></h4>
+      <p>+Review</p>
```

Comment:
I personally prefer just Review by itself
I think the icon conveys the meaning as well
```suggestion
      <p>Review</p>
```

Reply:
done